# single-cell_Analytical-Tools
single-cell analytical tools

## Gene Regulatory Networks in single-cells

### SCENIC: https://www.nature.com/articles/nmeth.4463, https://github.com/aertslab/SCENIC
#### SCENIC+: https://www.nature.com/articles/s41592-023-01938-4?utm_source=twitter&utm_medium=social&utm_campaign=nmeth

## Trajectory inference:

### Palantir: https://github.com/dpeerlab/Palantir
### Slingshot: https://nbisweden.github.io/workshop-archive/workshop-scRNAseq/2020-01-27/labs/compiled/slingshot/slingshot.html
### CellRank: https://cellrank.readthedocs.io/en/latest/

## Cell Annotation 

### CellTypist: https://github.com/Teichlab/celltypist
### SEACells: https://github.com/dpeerlab/SEACells


## Differential Abundance analysis:

### MILO: https://github.com/MarioniLab/miloR


## Ligand-receptor signaling

### LIANA: https://saezlab.github.io/liana/


## Bulk deconvolution

### BayesPrism: https://www.nature.com/articles/s43018-022-00356-3

## Reference Mapping:

### SCArches: https://github.com/theislab/scarches
### Azimuth: https://github.com/satijalab/azimuth
### SCimilarity: https://www.biorxiv.org/content/10.1101/2023.07.18.549537v1

## Density estimation
### MELLON: https://github.com/settylab/mellon

## Foundational Models
### scGPT: https://github.com/bowang-lab/scGPT
### Geneformer: https://www.nature.com/articles/s41586-023-06139-9
### SCimilarity

## VDJ Alignment Tools

### MiXCR: https://github.com/milaboratory/mixcr/releases/tag/v4.4.0
### TRUST4: https://www.nature.com/articles/s41592-021-01142-2
